docs/src/pages/customization/typography/typography.md

```diff
@@ -105,7 +105,12 @@ const theme = createTheme({
 
 The computed font size by the browser follows this mathematical equation:
 
-<img src="/static/images/font-size.svg" alt="font size calculation" style="width: 458px;" />
+<div class="only-light-mode">
+  <img src="/static/images/font-size.svg" alt="font size calculation" style="width: 458px;" />
+</div>
+<div class="only-dark-mode">
+  <img src="/static/images/font-size-dark.svg" alt="font size calculation" style="width: 458px;" />
+</div>
```

Comment:
This creates a layout shift when loading. Try https://mui.com/customization/typography/#font-size